src/databricks/labs/ucx/azure/credentials.py

```diff
@@ -298,9 +316,9 @@ def run(self, prompts: Prompts, include_names: set[str] | None = None) -> list[S
             "[RECOMMENDED] Please confirm to create an access connector with a managed identity for each storage "
             "account."
         )
-        ac_results = []
+        execution_results: list[StorageCredentialValidationResult] = []
```

Comment:
Rename is not consistent:
- sp_results -> service principal results
- ac_results -> access connector results
- execution_results = sp_results + ac_results

Reply:
done